config.js

```diff
@@ -539,6 +539,9 @@ config.BUCKET_REPLICATION_MAX_DST_BUCKETS = 100;
 config.REPLICATION_SEMAPHORE_CAP = 5000;
 config.REPLICATION_SEMAPHORE_TIMEOUT = 300 * 1000;
 config.REPLICATION_ENABLED = true;
+config.LOG_REPLICATION_ENABLED = true;
+config.AWS_LOG_CANDIDATES_LIMIT = 10;
+config.BUCKET_LOG_REPLICATOR_DELAY = 1 * 120 * 1000;
```

Comment:
I am not sure why this time was choses. I agree with the format as well. I think this is coming from Kfir's work on this as well. Both I and @MeridianExplorer did not know the rationale behind this number so we left it untouched I guess.

Having said that, the rationale could be that:
- Logs grow **really** quickly. Like **REALLY** quickly.
- We keep `HEAD`ing the objects in the bucket also adds the logs. I think if the delay is too much then we can take several hours to process just about 1000 logs (which can be generated really quickly). Say, we process 10 logs every 5 mins then we will take about 5 * 100 (i.e ~ 8 hours to process first 1000 logs).

Reply:
but we have a limit for the candidate's number also so we won't process 1000 log files, right? I wonder if running this bg worker every 2 minutes will make noobaa work harder, as you said we are heading every object every 2 minutes and it's added to the logs as well

Reply:
Oh no, I meant that let's say when we begin the processing the logs there are 1000 logs in the s3 bucket. Now we process 10 logs every 5 minute. So to churn through these 1k logs it will take us 5 * 100 minutes (~ 8 hours) and this churning will create more logs for us to process. 

For example say following sequence events happen:
1. User start replication via NooBaa while there are already 1k logs (not actually a lot - usually this amount of log is generated very quickly).
2. NooBaa starts processing the logs and this processing of 1k logs will take us about 8 hours.
3. While we were processing, user deleted `x` number of objects - this created more logs.

In such sequence of events, we will take at least 8 hours (no upper bound) to process the 2 deletes.

Having said that, frequent log churning will also be heavy on NooBaa but I am not sure what the right amount of delay would be.

Reply:
@dannyzaken thoughts about the correct delay?

Reply:
I don't think there was any deep discussion regarding this value. maybe we can change it to 5 like the scan-based and fine-tune it later